packages/in_app_purchase/in_app_purchase_storekit/lib/src/store_kit_wrappers/sk_payment_transaction_wrappers.dart

```diff
@@ -199,4 +219,20 @@ class SKPaymentTransactionWrapper {
         'transactionIdentifier': transactionIdentifier,
         'productIdentifier': payment.productIdentifier,
       };
+
+  /// Converts [SKPaymentTransactionMessages] into the dart equivalent
+  static SKPaymentTransactionWrapper convertFromPigeon(
+      SKPaymentTransactionMessage msg) {
+    return SKPaymentTransactionWrapper(
+        payment: SKPaymentWrapper.convertFromPigeon(msg.payment),
+        transactionState: SKPaymentTransactionStateWrapper.convertFromPigeon(
+            msg.transactionState),
+        originalTransaction: msg.originalTransaction == null
+            ? null
+            : convertFromPigeon(msg.originalTransaction!),
+        transactionTimeStamp: msg.transactionTimeStamp,
+        transactionIdentifier: msg.transactionIdentifier,
+        error:
+            msg.error == null ? null : SKError.convertFromPigeon(msg.error!));
```

Comment:
It's weird that you have use a `!` here, is this redundant?  Sometimes it seems like dart can't figure that out.

Reply:
the analyzer gets mad at me if i remove it lol